doc/guides/backport-to-lts-branches.md

```diff
@@ -0,0 +1,124 @@
+# How to backport to LTS branches
+
+### Set up your repo
+
+- Collaborator access
+
+- Make sure nobody else is currently backporting to the same LTS branch
+
+- Clone node checkout staging branch
+  ```bash
+  git clone git@github.com/nodejs/node --origin DANGER
```

Comment:
can you explain `--origin DANGER` rather than `upstream` which is usually the convention I've seen used

Reply:
That was a suggestion from @gibfahn for my setup - i'll swap it back to `upstream` so that it is consistent with other docs.